usr/src/uts/common/fs/zfs/sys/zap.h

```diff
@@ -87,22 +88,17 @@ extern "C" {
 
 /*
  * Specifies matching criteria for ZAP lookups.
+ * MT_EXACT - exact match, bypasses normalization to do strcmp()
+ * MT_FIRST - Normalizes the lookup - compose/decompose and then
+ *            match the normalized string.
+ * MT_CASE - Used to keep track of whether the lookup request is
```

Comment:
I'm certainly OK with changing the name to be more accurate (MT_CASE -> MT_CASE_SENSITIVE).  I'm still not sure I follow why the "sense" of the two bits has to be opposite.  For clarity it would be preferable if both bits indicated either "ignore this aspect" or "pay attention to this aspect".

I.e. either (MT_CASE_SENSITIVE, MT_FORM_SENSITIVE) or (MT_CASE_INSENSITIVE, MT_FORM_INSENSITIVE (or MT_NORMALIZED))